awscli/customizations/argrename.py

```diff
@@ -101,6 +101,11 @@
     'codepipeline.get-action-type.version': 'action-version',
     'ecs.*.no-enable-execute-command': 'disable-execute-command',
     'ecs.execute-command.no-interactive': 'non-interactive',
+    'mgn.*.replication-servers-security-groups-i-ds':
+        'replication-servers-security-groups-ids',
+    'mgn.*.source-server-i-ds': 'source-server-ids',
+    'mgn.*.replication-configuration-template-i-ds':
+        'replication-configuration-template-ids',
```

Comment:
If we cannot generalize this approach, we should move these to the `HIDDEN_ALIASES` dictionary as if we do not we would be introducing a breaking change since the poorly named id parameters would be removed and users could be writing scripts that use these poorly named parameters because they are the only option.